packages/storage/src/cookies.ts

```diff
@@ -120,28 +129,31 @@ export const cookieStorage: StorageWithOptions<CookieOptions> = addClearMethod({
       window.dispatchEvent(storageEvent);
     }
   },
-  removeItem: (key: string) => {
-    cookieStorage._write(key, "deleted", {expires: new Date(0)});
+  removeItem: (key: string, options?: CookieOptions) => {
+    cookieStorage._write(key, "deleted", {...options, expires: new Date(0)});
   },
-  key: (index: number) => {
+  key: (index: number, options?: CookieOptions) => {
     let key: string | null = null;
     let count = 0;
-    cookieStorage._read().replace(/(?:^|;)\s*(.+?)\s*=\s*[^;]+/g, (_: string, found: string) => {
+    cookieStorage._read(options).replace(/(?:^|;)\s*(.+?)\s*=\s*[^;]+/g, (_: string, found: string) => {
       if (!key && found && count++ === index) {
         key = found;
       }
       return "";
     });
     return key;
   },
-  get length() {
+  getLength: (options?: CookieOptions) => {
```

Comment:
Why would that be better than the previous solution?

Reply:
what do you mean exactly, i didn't really thouch the code, but i had to add the CookieOptions for each method, because instead you won't be able to pass the getRequest method

Reply:
I think we should rethink the whole API, but it will be a breaking change. We should have a init(options) call so everything else will work like normal storage API.

Reply:
yeah, but this is in my opinion for another pull request ...  i just want to get an release with this pull request so that my colleges don't have to download and compile my fork each time :)